config/lava/kselftest/kselftest.jinja2

```diff
@@ -21,6 +21,6 @@
       name: {{ plan }}
       parameters:
         TESTPROG_URL: {{ kselftests_url }}
-        SKIPFILE: skipfile-lkft.yaml
```

Comment:
I think we should have a default value defined somewhere, for the kselftest variants that don't define `kselftest_skipfile`.  Maybe this could be done in the template:
```yaml
SKIPFILE: {{ kselftest_skipfile|default("skipfile-lkft.yaml", true) }}
```